src/flows/components/panel/FlowPanel.tsx

```diff
@@ -100,6 +102,7 @@ const FlowPanelHeader: FC<HeaderProps> = ({id, controls}) => {
               active={canBeMovedDown}
             />
           </FeatureFlag>
+          {shouldShow && <ExportTaskButton />}
```

Comment:
this should turn into whatever that shouldShow turns into. Follow the `controls` pattern here, just that it's not hidden. And for next time, if you find yourself making panel specific changes is the flows pattern, and not just on the pipe, that should shoot off some red flags